x-pack/plugins/ml/public/components/field_title_bar/_field_title_bar.scss

```diff
@@ -1,9 +1,10 @@
 .ml-field-title-bar {
   color: $euiColorEmptyShade;
   font-size: $euiFontSizeL;
+  line-height: $euiSizeL;
```

Comment:
This works, but `@include euiFontSizeL` will give you size and line height together in a more stable format that's more vertical rhythm aware. It should even remove your need for the padding (didn't test).

https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_typography.scss#L81-L84

Reply:
Thanks for the suggestion @snide. I've switched out the `font-size` and `line-height` for `@include euiFontSizeL`. Opted to retain just `padding-bottom` after testing.